addons/mail/models/mail_channel.py

```
@@ -1133,3 +1133,13 @@ def _execute_command_who(self, **kwargs):
             msg = _("Users in this channel: %(members)s %(dots)s and you.", members=", ".join(members), dots=dots)
 
         self._send_transient_message(partner, msg)
+
+    @api.model
+    def _mark_seen_general_channel(self):
+        channel = self.env.ref('mail.channel_all_employees', raise_if_not_found=True)
+        last_message_id = channel._channel_last_message_ids()
+        if channel and last_message_id:
```

no need to make `if channel` if you raise if it's not found